releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1774,7 +1832,7 @@
   "124500": {
     "commit": "8240d882ab614d40538c7e441226be33bbdfb021",
     "text": "kube-scheduler deprecated all non-csi volumelimit plugins and removed those from defaults plugins. \n- AzureDiskLimits\n- CinderLimits\n- EBSLimits\n- GCEPDLimits\n\nThe NodeVolumeLimits plugin can handle the same functionality as the above plugins since the above volume types are migrated to CSI.\nPlease remove those plugins and replace them with the NodeVolumeLimits plugin if you explicitly use those plugins in the scheduler config.\nThose plugins will be removed in the release 1.32.",
-    "markdown": "Kube-scheduler deprecated all non-csi volumelimit plugins and removed those from defaults plugins. \n  - AzureDiskLimits\n  - CinderLimits\n  - EBSLimits\n  - GCEPDLimits\n  \n  The NodeVolumeLimits plugin can handle the same functionality as the above plugins since the above volume types are migrated to CSI.\n  Please remove those plugins and replace them with the NodeVolumeLimits plugin if you explicitly use those plugins in the scheduler config.\n  Those plugins will be removed in the release 1.32. ([#124500](https://github.com/kubernetes/kubernetes/pull/124500), [@carlory](https://github.com/carlory)) [SIG Scheduling and Storage]",
+    "markdown": "Kube-scheduler deprecated all non-csi volumelimit plugins and removed those from defaults plugins. \n  - AzureDiskLimits\n  - CinderLimits\n  - EBSLimits\n  - GCEPDLimits\n  \n  The NodeVolumeLimits plugin can handle the same functionality as the above plugins since the above volume types are migrated to CSI.\n  Please remove those plugins and replace them with the NodeVolumeLimits plugin if you explicitly use those plugins in the scheduler config.\n  Those plugins will be removed in the release 1.32. ([#124500](https://github.com/kubernetes/kubernetes/pull/124500), [@carlory](https://github.com/carlory))",
```

Comment:
```suggestion
    "markdown": "Kube-scheduler deprecated all non-csi volumelimit plugins and removed those from defaults plugins. \n  - AzureDiskLimits\n  - CinderLimits\n  - EBSLimits\n  - GCEPDLimits\n  \n  The NodeVolumeLimits plugin can handle the same functionality as the above plugins since the above volume types are migrated to CSI.\n  Please remove those plugins and replace them with the NodeVolumeLimits plugin if you explicitly use those plugins in the scheduler config.\n  Those plugins will be removed in the release 1.32. ([#124500](https://github.com/kubernetes/kubernetes/pull/124500), [@carlory](https://github.com/carlory)) [SIG Scheduling and Storage]",
```